discord/cog.py

```diff
@@ -23,7 +23,8 @@
 DEALINGS IN THE SOFTWARE.
 """
 from __future__ import annotations
-
+import os
+import pathlib
 import importlib
 import inspect
 import sys
```

Comment:
Alphabetize the newly imported modules.
```suggestion
import importlib
import inspect
import os
import pathlib
import sys
```